src/server_manager/web_app/app.ts

```diff
@@ -145,6 +147,9 @@ export class App {
       private digitalOceanTokenManager: TokenManager, private surveys: Surveys) {
     appRoot.setAttribute('outline-version', this.version);
 
+    const shareApp = new ShareDialogApp(appRoot.$.shareDialog);
+    const getConnectedApp = new GetConnectedApp(appRoot.$.getConnectedDialog);
```

Comment:
nit: `s/App/Controller`. I think app is not very intuitive to understand the relationship of the class with the view element.